QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -319,10 +283,10 @@ def convert_to_cloud_kvstyle(
 
     # Run PyTorch inference with past
     try:
-        pt_outputs = model(**inputs)
+        pt_outputs = transformed_model(**inputs)
         output_names = list(pt_outputs.keys())
     except Exception as e:
-        print(f"Model {model_name} Execution failed in pytorch:%s", e)
+        print(f"Model {transformed_model.__class__,__name__} Execution failed in pytorch:%s", e)
```

Comment:
Should not we break the code with exception? Continuing post exception at torch doesn't make sense.
`raise RuntimeError()`?

Reply:
removing the try-except block.